packages/svelte/src/internal/server/blocks/svelte-html.js

```diff
@@ -0,0 +1,13 @@
+/** @import { Payload } from '#server' */
+
+import { escape_html } from '../../../escaping.js';
+
+/**
+ * @param {Payload} payload
+ * @param {Record<string, string>} attributes
+ */
+export function svelte_html(payload, attributes) {
+	for (const name in attributes) {
+		payload.htmlAttributes.set(name, escape_html(attributes[name], true));
```

Comment:
what if the attribute already exists? do we error, or overwrite?

what if it's a `class` attribute? e.g. you have one component like this...

```svelte
<svelte:html class="{theme === 'dark' ? 'dark' : undefined}" />
```

...and another like this?

```svelte
<svelte:html class="{prefers_typescript ? 'prefers-ts' : 'prefers-js'}" />
```

Neither behaviour would be desirable in that case. Maybe `class` needs to be treated as a special case?

Reply:
Currently it's "last one wins" for all attributes, similar to how the last `<title>` would win if you had multiple of them within `<svelte:head>` - which I think makes the most sense. Agree that `class` could use special handling though.

Reply:
Should we at least warn if attributes are getting clobbered? Seems more likely to be a mistake than anything. (The same is probably true of `<title>`, and possibly other unique head tags, though we don't need to solve that right now)